classes/webHookDispatcher/OrderDispatcher.php

```diff
@@ -55,20 +55,22 @@ public function __construct()
     public function dispatchEventType($payload)
     {
         $psOrderId = $this->getPrestashopOrderId($payload['orderId']);
+        // Some webhook (queued) sent json encoded data
+        $resource = is_array($payload['resource']) ? $payload['resource'] : \Tools::jsonDecode($payload['resource'], true);
```

Comment:
Note you don't have to use anymore `Tools::jsonEncode` and you're free to use the native method `json_encode`

Reply:
With PrestaShop 1.6 too ?

Reply:
On all PrestaShop versions, this has been removed from the validation rules.

Reply:
I will do it in another PR, there are many `Tools::jsonDecode()`elsewhere